metricbeat/module/prometheus/prometheus_pb2.py

```diff
@@ -0,0 +1,577 @@
+# -*- coding: utf-8 -*-
```

Comment:
is this file needed?

Reply:
It is required by the python tests: https://github.com/elastic/beats/pull/16609/files#diff-f0fbea18a4681afd920cb91b903d03d2R6